Remotion/Development/UnitTesting.IsolatedCodeRunner/Development.UnitTesting.IsolatedCodeRunner.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectType>Executable</ProjectType>
+    <TargetFrameworks></TargetFrameworks>
+    <TargetFramework>net6.0-windows</TargetFramework>
```

Comment:
Same as in Remotion/Development/UnitTesting.IsolatedCodeRunner.UnitTests/Development.UnitTesting.IsolatedCodeRunner.UnitTests.csproj